### PR TITLE
Add cd in the instructions - just for quick copy/paste

### PR DESCRIPTION
**Pull request by valeriabarra**
My very first and very quick PR! 

### Description

This just adds the code snippet for the "change directory" instruction, that was previously expressed in words. I found myself doing just quick copy&paste of the code parts, and if someone doesn't note the written part, this wouldn't work. So it's just to allow easy reproducibility of the instructions.

- [ ] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [ ] Unit tests are included OR N/A.
- [ ] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: docs/src/GettingStarted/Installation.md [hunk 1]
```diff
@@ -44,7 +44,13 @@ Download the `ClimateMachine`
...
 
-Now change into the `ClimateMachine.jl` directory and install all the packages
+Now change into the `ClimateMachine.jl` directory with 
+
+```
+$ cd ClimateMachine.jl
+```
+
+and install all the packages
 required with:
```
Comment (charleskawczynski) [inline docs/src/GettingStarted/Installation.md:54]:
```suggestion
and install all the required packages with:
```

**Review by charleskawczynski — APPROVED** [review]
LGTM, let's squash and merge!

**Review by jkozdon — APPROVED** [review]

**Comment by valeriabarra** [comment]
Bors, please don't hate me already 😭 it was nice to meet you

**Pull request merged**